website/src/theme/Feedback/AddFeedback.tsx

```diff
@@ -36,41 +36,96 @@ class AddFeedback extends React.Component<IProps, IState> {
     };
   }
 
+  async postToSlack(rating?: number) {
+    const url = process.env.SLACK_WEBHOOK_URL;
```

Comment:
Guessing i'll need to add this to my local .env file after this is merged and also cloudflare deploy configuration?